WebContent/js/utilities/sharedUtils.js

```diff
@@ -0,0 +1,20 @@
+/*
+* Each name segment (qualifier) is 1 to 8 characters, the first of which must be alphabetic (A to Z) or national (# @ $).
```

Comment:
You should link https://www.ibm.com/docs/en/zos/2.1.0?topic=sets-data-set-names in the comments here

Reply:
Done